packages/events/src/handlers/target-removed.ts

```diff
@@ -0,0 +1,29 @@
+import type { TargetRemoved } from "@ctrlplane/validators/events";
+
+import { and, eq } from "@ctrlplane/db";
+import { db } from "@ctrlplane/db/client";
+import * as SCHEMA from "@ctrlplane/db/schema";
+import { dispatchRunbook } from "@ctrlplane/job-dispatch";
+
+export const handleTargetRemoved = async (event: TargetRemoved) => {
+  const { target, deployment } = event.payload;
+
+  const isSubscribedToTargetRemoved = and(
+    eq(SCHEMA.hook.scopeId, deployment.id),
+    eq(SCHEMA.hook.scopeType, "deployment"),
+    eq(SCHEMA.hook.event, "target.removed"),
```

Comment:
```suggestion
    eq(SCHEMA.hook.event, "target"),
    eq(SCHEMA.hook.action, "removed"),
```

is what I had in mind